contentcuration/contentcuration/models.py

```diff
@@ -268,6 +346,90 @@ def _save(self, name, content):
         return super(FileOnDiskStorage, self)._save(name, content)
 
 
+@deconstructible
+class ObjectStorage(Storage):
+    """
+    ObjectStorage stores our data in Minio, an object storage system that's Amazon S3 compatible.
+
+    Minio runs on your local machine, allowing developers to emulate S3. In production, Minio acts as a proxy
+    to a production storage system, such as Google Cloud Storage, or the real Amazon S3.
```

Comment:
Sorry since this is a lot of change, I'm wondering that in production, when we need to add the files to Google Cloud Storage, which files are setting that up? I thought we might need google credentials to do it but didn't find a variable for it in settings. Thank you!

Reply:
@lyw07 we just need to switch over the API, access ID and access key over to GCS, and we should be good!